Index: PHP/JSON/API/Api_example.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/PHP/JSON/API/Api_example.php b/PHP/JSON/API/Api_example.php
new file mode 100644
--- /dev/null	(revision bdc5d647ecb267e6a3699a90801815207f1dc42e)
+++ b/PHP/JSON/API/Api_example.php	(revision bdc5d647ecb267e6a3699a90801815207f1dc42e)
@@ -0,0 +1,5 @@
+<?php
+$url = "";
+$data= file_get_contents($url);
+$characters = json_decode($data);
+print_r($characters);
\ No newline at end of file
